src/components/drawing/index.js

```diff
@@ -112,6 +112,7 @@ drawing.lineGroupStyle = function(s, lw, lc, ld) {
 };
 
 drawing.dashLine = function(s, dash, lineWidth) {
+    lineWidth = +lineWidth || 0;
```

Comment:
Interesting. Maybe this also fixes https://github.com/plotly/plotly.js/issues/166 ?